trillian/ctfe/configpb/config.proto

```diff
@@ -132,6 +133,33 @@ message LogConfig {
   // A list of X.509 extension OIDs, in dotted string form (e.g. "2.3.4.5")
   // which should cause submissions to be rejected.
   repeated string reject_extensions = 18;
+
+  // CTFE storage connection string in the following format in general:
+  // driver://[username[:password]@][protocol[(host[:port])]][/[schema|database][?options]]
+  //
+  // MySQL/MariaDB:
+  // mysql://[username[:password]@][protocol[(address)]]/dbname[?param1=value1&...&paramN=valueN]
+  //
+  // This is required when the non-leaf certificate chain storage backend is
+  // CTFE.
+  // 
+  // Warning: CT log operators are advised not to share the non-leaf 
```

Comment:
To avoid any interpretation ambiguity, be explicit: do not re-use the same connection string across multiple LogConfigs.

Reply:
Done.